<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerLight</class>
 <widget class="QWidget" name="ManagerLight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager Light</string>
  </property>
  <widget class="QRadioButton" name="Flat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Flat</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="Gouraud">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Gouraud</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Phong">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>10</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Phong</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Ambiental</string>
    </attribute>
    <widget class="QCheckBox" name="activeAmbienta1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>74</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Activo</string>
     </property>
    </widget>
    <widget class="QLabel" name="colorAmb1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="chColorAmb1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="colorDiff">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="matDiff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Color Especular</string>
     </property>
    </widget>
    <widget class="QLabel" name="colorEsp">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="matEsp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Color Especular</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Difusa 1</string>
    </attribute>
    <widget class="QCheckBox" name="activeDifusa1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>74</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Activo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos. X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos. Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos. Z</string>
     </property>
    </widget>
    <widget class="QSlider" name="posAmbX3">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>30</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="ledPosX3">
     <property name="geometry">
      <rect>
       <x>176</x>
       <y>120</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="ledPosY3">
     <property name="geometry">
      <rect>
       <x>229</x>
       <y>120</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="colorDif1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="posAmbY3">
     <property name="geometry">
      <rect>
       <x>236</x>
       <y>30</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="chColorDif1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QSlider" name="posAmbZ3">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>30</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="ledPosZ3">
     <property name="geometry">
      <rect>
       <x>277</x>
       <y>120</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Difusa 2</string>
    </attribute>
    <widget class="QCheckBox" name="activeDifusa2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>74</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Activo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos. X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos. Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos. Z</string>
     </property>
    </widget>
    <widget class="QSlider" name="posAmbX4">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>30</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="ledPosX4">
     <property name="geometry">
      <rect>
       <x>176</x>
       <y>120</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="ledPosY4">
     <property name="geometry">
      <rect>
       <x>229</x>
       <y>120</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="colorDif2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="posAmbY4">
     <property name="geometry">
      <rect>
       <x>236</x>
       <y>30</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="chColorDif2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QSlider" name="posAmbZ4">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>30</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="ledPosZ4">
     <property name="geometry">
      <rect>
       <x>277</x>
       <y>120</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Especular 1</string>
    </attribute>
    <widget class="QCheckBox" name="activeEspecular1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>74</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Activo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos. X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos. Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos. Z</string>
     </property>
    </widget>
    <widget class="QSlider" name="posAmbX5">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>30</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="ledPosX5">
     <property name="geometry">
      <rect>
       <x>176</x>
       <y>120</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="ledPosY5">
     <property name="geometry">
      <rect>
       <x>229</x>
       <y>120</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="colorEsp1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="posAmbY5">
     <property name="geometry">
      <rect>
       <x>236</x>
       <y>30</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="chColorEsp1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QSlider" name="posAmbZ5">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>30</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="ledPosZ5">
     <property name="geometry">
      <rect>
       <x>277</x>
       <y>120</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Especular 2</string>
    </attribute>
    <widget class="QCheckBox" name="activeEspecular2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>74</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Activo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos. X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos. Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pos. Z</string>
     </property>
    </widget>
    <widget class="QSlider" name="posAmbX6">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>30</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="ledPosX6">
     <property name="geometry">
      <rect>
       <x>176</x>
       <y>120</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="ledPosY6">
     <property name="geometry">
      <rect>
       <x>229</x>
       <y>120</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="colorEsp2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="posAmbY6">
     <property name="geometry">
      <rect>
       <x>236</x>
       <y>30</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="chColorEsp2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QSlider" name="posAmbZ6">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>30</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-250</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="ledPosZ6">
     <property name="geometry">
      <rect>
       <x>277</x>
       <y>120</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>10</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver puntos</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="shininess">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>7</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>128</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>9</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Shininess</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
